lib/ClangImporter/ImportName.cpp

```diff
@@ -1597,11 +1597,11 @@ static bool shouldIgnoreMacro(StringRef name, const clang::MacroInfo *macro) {
   if (macro->tokens_empty())
     return true;
 
-  // Currently we only convert non-function-like macros.
-  if (macro->isFunctionLike())
+  // Currently we only convert function-like macros that take zero arguments.
+  if (macro->isFunctionLike() && macro->getNumArgs() != 0)
     return true;
 
-  // Consult the blacklist of macros to suppress.
+  // Consult the list of macros to suppress.
```

Comment:
"Blacklist" is considered exclusionary in some contexts. Here it seems "list" conveys the same meaning without any unintended consequences. I was already modifying the body of this method, so I thought I'd include this comment change as well.